Formula/vcluster.rb

```diff
@@ -0,0 +1,35 @@
+class Vcluster < Formula
+  desc "Creates fully functional virtual k8s cluster inside host k8s cluster's namespace"
+  homepage "https://www.vcluster.com"
+  url "https://github.com/loft-sh/vcluster.git",
+    tag:      "v0.7.1",
+    revision: "dc0ff6f96e9c96fe2caa77e79c2dffc921b4fd49"
+  license "Apache-2.0"
+  head "https://github.com/loft-sh/vcluster.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "helm"
+  depends_on "kubernetes-cli"
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.commitHash=#{Utils.git_head}
+      -X main.buildDate=#{time.iso8601}
+      -X main.version=#{version}
+    ]
+    system "go", "build", "-mod", "vendor", *std_go_args(ldflags: ldflags), "./cmd/vclusterctl/main.go"
+  end
+
+  test do
+    help_output = "vcluster root command"
+    assert_match help_output, shell_output("#{bin}/vcluster --help")
+
+    create_output = "there is an error loading your current kube config "\
+                    "(invalid configuration: no configuration has been provided, "\
+                    "try setting KUBERNETES_MASTER environment variable), "\
+                    "please make sure you have access to a kubernetes cluster and the command "\
+                    "`kubectl get namespaces` is working"
```

Comment:
```suggestion
    create_output = <<~EOS
      there is an error loading your current kube config 
      (invalid configuration: no configuration has been provided, 
      try setting KUBERNETES_MASTER environment variable), 
      please make sure you have access to a kubernetes cluster and the command 
      `kubectl get namespaces` is working
    EOS
```
This is a bit cleaner, but `brew` might complain about the trailing spaces.